include/dungeon_map_access.h

```diff
@@ -8,10 +8,10 @@ typedef struct unkStruct_202F190
     Tile *unk0[6];
 } unkStruct_202F190;
 
-Tile *GetTile(s32 x, s32 y);
-Tile *GetTileSafe(s32 x, s32 y);
+const Tile *GetTile(s32 x, s32 y);
```

Comment:
I know there was a question on discord the difference between the two. Might be worth copying/pasting the explanation here?

Reply:
I feel like the types - one returning a const pointer and one returning a regular one - speak for themselves. Not sure what kind of explanation I could otherwise put here.